### ISSUE TITLE
[Issue] About when submitting a map and leaving the tab

### ISSUE THREAD

**Issue by DragoBruder** [ppy/osu-stable-issues#608]
**Hey,

I experienced a "bug" or rather an issue that almost ended a whole map.

I wrote an email so I will just copy and paste the email here with all the information.**

Hey osu!team, (or only peppy I really don’t know)

when I was About to submit/update my map, I almost deleted a lot of effort in a few seconds.

The Problem I Mention here is, when you submit a map you may actually press space bar or something (in my case the entire desciption was marked) and it’s all gone.

I thought like: Oh shit. What now??? Uhm… Well, press the „X“ I guess.

Well funny Thing here.

That „X“ that was supposed to deny any changes and Close the submitting tab almost ended my whole map actually. 

Basically I pressed spacebar by accident and I deleted the description, I wanted to deny submitting that and pressed the „X“. That resulted in submitting the map actually.

Good Thing About that, I had a tab open with my map where I can copy&paste the description back. But if I wouldn’t have the tab, it really would fuck me up hard.

Is it possible to Change that? That the „X“ actually does deny the submitting process?

**Comment by peppy** [comment]
It's quite hard to change this logic. The submit-on-close is there as a safety measure due to the convoluted way beatmap submission works in stable.

We'll definitely make sure this isn't a thing in lazer. I'd recommend keeping a browser tab open if you are worried about this happening (or contact us immediately after you lose content so we can restore it for you).
